data/national_secondary_admissions_and_live_births.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  apps_offers.year,
+  apps_offers.admission_numbers,
+  apps_offers.applications_received,
+  (apps_offers.first_preference_offers / apps_offers.applications_received) * 100 as percentage_first_pref,
```

Comment:
I think there's a percent function in SQL

Reply:
I can't find one!